sonic-net/sonic-buildimage
# chore: add 202405 sonic release file

#### Why I did it
On 202405, `release: 'none'` appears in the `/etc/sonic/sonic_version.yml` file

##### Work item tracking
- Microsoft ADO **(number only)**:

#### How I did it
Add a `sonic_release` file to `sonic-buildimage/files/image_config/`, just like what we did before: https://github.com/sonic-net/sonic-buildimage/pull/9985, https://github.com/sonic-net/sonic-buildimage/pull/16364

#### How to verify it
Install the image and run: `cat /etc/sonic/sonic_version.yml`, then verify the following item in the `/etc/sonic/sonic_version.yml` file: `release: '202405'`



#### Which release branch to backport (provide reason below if selected)




#### Tested branch (Please provide the tested image version)




#### Description for the changelog




#### Link to config_db schema for YANG module changes


#### A picture of a cute animal (not mandatory but encouraged)
